Deploy-Bastion.ps1

```diff
@@ -0,0 +1,115 @@
+param
+(
+    [ValidateNotNullOrEmpty()]
+    [Parameter(Mandatory=$true, HelpMessage="The Name of the new DevTest Lab")]
+    [string] $DevTestLabName,
+
+    [ValidateNotNullOrEmpty()]
+    [Parameter(Mandatory=$true, HelpMessage="The Name of the resource group")]
+    [string] $ResourceGroupName,
+
+    [parameter(Mandatory=$false, HelpMessage="Public=separate IP Address, Shared=load balancers optimizes IP Addresses, Private=No public IP address.")]
+    [string] $LabIpConfig = "Public",
```

Comment:
It doesn't look like $LabIpConfig is used in this script (unless I missed it).  If not, you can remove from the parameters...